pkg/application/app.go

```diff
@@ -180,3 +181,15 @@ func (app *Avalanche) ChainIDExists(chainID string) (bool, error) {
 
 	return false, nil
 }
+
+func (app *Avalanche) LoadNodeConfig() (string, error) {
```

Comment:
I would probably make a config object with one field for each needed config item, create it in initConfig, and them pass it to app, which will expose it, eg app.Conf.GlobalNodeConf

Reply:
I made the config object as you suggested